modules/cnv-confirming-policy-updates-on-nodes.adoc

```diff
@@ -0,0 +1,39 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_node_network/cnv-configuring-node-network-policy
+
+[id="cnv-confirming-policy-updates-on-nodes_{context}"]
+= Confirming Policy updates on nodes
+
+A `NodeNetworkConfigurationPolicy` manifest describes your desired network configuration for nodes in the cluster.
+When applied, the cluster creates a `NodeNetworkConfigurationPolicy` object for the whole cluster.
```

Comment:
The object is created by user, not by the cluster. I think we can drop this line. Users can tell they need to apply it and what happens after that is obvious from the previous sentence and the rest of the document.